moped-editor/src/queries/components.js

```diff
@@ -97,6 +99,30 @@ export const GET_PROJECT_COMPONENTS = gql`
       component_id
       attributes
     }
+    parentProjectComponents: moped_proj_components(
+      where: {
+        project_id: { _eq: $parentProjectId }
+        is_deleted: { _eq: false }
+      }
+    ) {
+      ...projectComponentFields
+    }
+    siblingProjects: moped_project(
+      where: { parent_project_id: { _eq: $parentProjectId } }
```

Comment:
do we need to include where is_deleted is false here as well? and in the childProjects part of the query

Reply:
Oh yes! Awesome catch, will add this in along with Patrick's suggestion below. 🙏